dist/pathfora.js

```diff
@@ -2016,7 +2091,43 @@
           }
         });
       });
-    }
+    },
+
+
+    /**
+     * @description Fetch content to recommend
+     * @throws {Error} error
+     * @param {string} accountId  Lytics account ID
+     */
+    recommendContent: function (accountId, filter, callback) {
+      var seerId = utils.readCookie('seerid');
+      var recommendUrl;
+
+      if (!seerId) {
+        // set a default here, instead
```

Comment:
need this comment still? dont think we want pathfora changing the seerid since thats a function of jstag.